event.go

```diff
@@ -14,11 +14,6 @@ type EventTag struct {
 	Value interface{}
 }
 
-// EventBaggage is received when SetBaggageItem is called.
-type EventBaggage struct {
```

Comment:
I got rid of this since it was annoying to implement... if you care, it'd be easy enough to add back.